### PR TITLE
Dannym Minor Version Bump

### PR DESCRIPTION
**Pull request by dannymeloy**
Minor version bump due to the number of changes:
- Change scripts to run on just python3 due to deprecation
- Move an env. var from the bin file into the service file (it appeared to flag an error even when using https??)
- Fix error when using the client credentials grant due to authlib returning user=None
- Add password checking when requesting a code using the auth code grant
- Change 'iss' claim to give FQDN of auth server
- Change 'aud' claim to give a generic return value based on the scope (this should be changed but is better than what we had)
- Unpinned deb dependencies as is useful to have the systemd and apache debhelpers when installing using ansible
- Pass auth code params in query string as opposed to body of request
- Merge a couple of unittests in for registering a user and an OAuth client

### CODE REVIEW

File: nmosauth/auth_server/db_utils.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from six import string_types
```
Comment (andrewbonney) [inline nmosauth/auth_server/db_utils.py]: Is this service python 3 only, I can't remember. If so, can we avoid using six?

File: tests/test_integration.py [hunk 2]
```diff
@@ -92,23 +99,57 @@ def testBasicAuthRoutes(self):
...
+
         mockUser.query.filter_by.return_value.first.return_value = None
         with self.client.post(VERSION_ROOT + '/home/', data=dict(username="steve", password="password")):
             mockTemplate.assert_called_with(
                 'home.html', clients=None, message='That username is not recognised. Please signup.', user=None)
 
-    @mock.patch("nmosauth.auth_server.security_api.authorization")
-    def testTokenEndpoint(self, mockAuthServer):
-        mockAuthServer.create_endpoint_response.return_value = BEARER_TOKEN
+    def testRegisterClient(self):
```
Comment (JamesGibo) [inline tests/test_integration.py]: Just a style comment, most of the other repos with test use `lowercase_separated_by_underscores` for the test methods. Can't see anything in the ways of working to specify a preference.

Comment (dannymeloy) [reply tests/test_integration.py]: Yeah they should be underscores / snake_case really according to PEP guidelines...

File: nmosauth/auth_server/db_utils.py (hunk 1 above)
Comment (dannymeloy) [reply nmosauth/auth_server/db_utils.py]: It still supports python2 yeah, the certs would just need to be generated manually now I've changed the shebang lines of the script.

**Pull request merged**